scripts/build-index.js

```diff
@@ -29,8 +29,19 @@ function buildPagesIndex(files) {
       if (!index[page].language.includes(language)) {
         index[page].language.push(language);
       }
+
+      const targets = index[page].targets;
+      const exist = targets.some((t) => {return t.platform === os && t.language === language});
```

Comment:
No need for `()`, `{}` and `return` here. It can just be `t ⇒ expr`. Just checking if this is intended, if you want to keep it like this it works as well, it's just longer.

Reply:
Also, the variable should be called `exists`.

Reply:
Yeah, it is intended. I find the `()` to be more readable. Will change the variable.